kolibri/plugins/slideshow_viewer/assets/src/views/SlideshowRendererComponent.vue

```diff
@@ -121,6 +121,15 @@
       contentHeight: function() {
         return window.innerHeight * 0.7 + 'px';
       },
+      /* eslint-disable kolibri/vue-no-unused-properties */
+      /**
+       * @public
+       * Note: the default duration historically for slidshows has been 5 min
+       */
+      defaultDuration() {
+        return this.duration || 300;
```

Comment:
I was assuming that `defaultDuration` could just return a single value, and then the mixin can worry about conditionalizing between `duration` and the `defaultDuration`.

Reply:
🤦🏽‍♀️ You're right. The mixin already does that! Thanks for catching.